src/data/markdown/docs/01 guides/02 Using k6/14 Scenarios/04 Arrival Rate.md

```diff
@@ -0,0 +1,121 @@
+to finish of the VU's `exec` function, by default the `export default function`):
+
+<div class="code-group" data-props='{"labels": [ "closed-model.js" ], "lineNumbers": "[true]"}'>
+
+```js
+import http from 'k6/http';
+
+export let options = {
+  scenarios: {
+    closed_model: {
+      executor: 'constant-vus',
+      vus: 1,
+      duration: '1m',
+    },
+  },
+};
+
+export default function () {
+  // The following request will take roughly 6s to complete,
+  // resulting in an iteration duration of 6s.
+
+  // As a result, New VU iterations will start at a rate of 1 per 6s,
+  // and we can expect to get 10 iterations completed in total for the
+  // full 1m test duration.
+
+  http.get('http://httpbin.test.k6.io/delay/6');
+}
+```
+
+</div>
+
+Running this script would result in something like:
+
+```bash
+
+running (1m01.5s), 0/1 VUs, 10 complete and 0 interrupted iterations
+closed_model ✓ [======================================] 1 VUs  1m0s
+
+```
+
+## Drawbacks of using the closed model
+
+This tight coupling between the VU iteration duration and start of new VU iterations
+in effect means that the target system can influence the throughput of the test, via
+its response time. Slower response times means longer iterations and a lower arrival
+rate of new iterations, and vice versa for faster response times.
+
+In other words, when the target system is being stressed and starts to respond more
+slowly a closed model load test will play "nice" and wait, resulting in increased
+iteration durations and a tapering off of the arrival rate of new VU iterations.
+
+This is not ideal when the goal is to simulate a certain arrival rate of new VUs,
+or more generally throughput (eg. requests per second).
+
+## Open model
+
+To fix this problem we use an open model, decoupling the start of new VU iterations
+from the iteration duration and the influence of the target system's response time.
+
+![Arrival rate closed/open models](images/Scenarios/arrival-rate-open-closed-model.png)
+
+In k6 we've implemented this open model with our "arrival rate" executors. There are  
+two arrival rate executors to chose from for your scenario(s),
+[constant-arrival-rate](#constant-arrival-rate) and [ramping-arrival-rate](#ramping-arrival-rate):
+
+<div class="code-group" data-props='{"labels": [ "open-model.js" ], "lineNumbers": "[true]"}'>
+
+```js
+import http from 'k6/http';
+
+export let options = {
+  scenarios: {
+    open_model: {
+      executor: 'constant-arrival-rate',
+      rate: 1,
+      timeUnit: '1s',
+      duration: '1m',
+      preAllocatedVUs: 20,
+      maxVUs: 100,
+    },
+  },
+};
+
+export default function () {
+  // With the open model arrival rate executor config above,
+  // new VU iteration will start at a rate of 1 every second,
+  // and we can thus expect to get 60 iterations completed
+  // for the full 1m test duration.
+  http.get('http://httpbin.test.k6.io/delay/10');
```

Comment:
I would use `http://httpbin.test.k6.io/delay/6` to show exactly the same function example as before.